frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -126,18 +130,33 @@ const PlayersList = (props: PlayersListProps) => {
   );
 };
 
-const PlayerNotFound = () => (
-  <Centered>
-    <Alert severity="info">
-      <AlertTitle>Nie znaleziono zawodnika?</AlertTitle>
-      Zapisz nowego poniżej.
-    </Alert>
-    <VerticalSpace height="1rem" />
-    <Button variant="contained" color="primary">
-      Dodaj i zapisz
-    </Button>
-  </Centered>
-);
+const PlayerNotFound = () => {
+  const [drawerOpened, setDrawerOpened] = useState<boolean>(false);
+
+  const toggleDrawer = (open: boolean) => () => {
+    setDrawerOpened(open);
+  };
+
+  return (
+    <Centered>
+      <Alert severity="info">
+        <AlertTitle>Nie znaleziono zawodnika?</AlertTitle>
+        Zapisz nowego poniżej.
+      </Alert>
+      <VerticalSpace height="1rem" />
+      <Button variant="contained" color="primary" onClick={toggleDrawer(true)}>
+        Dodaj i zapisz
+      </Button>
+      <Drawer
+        anchor={"bottom"}
+        open={drawerOpened}
+        onClose={toggleDrawer(false)}
+      >
+        <AddingPlayerForm />
+      </Drawer>
+    </Centered>
+  );
+};
```

Comment:
To jutro ogarnę

Reply:
Drawera z tego dodawałem :-) 
https://www.youtube.com/watch?v=sFImlwEV5Rk&ab_channel=Jesus%2CTaketheCompiler